### PR TITLE
Strata random arms

### PR DESCRIPTION
**Pull request by Buhle79**
Changed api to pass mom age and facility code

### CODE REVIEW

File: mqr/utils.py [hunk 1]
```diff
@@ -70,3 +71,49 @@ def get_next_send_date():
...
+        clinic_code = None
+
+    return clinic_code
+
+
+def get_weeks_pregnant(edd):
+    full_term_weeks = 40
+    estimated_date = edd
+
+    if isinstance(edd, str):
```
Comment (erikh360) [inline mqr/utils.py]: I think if we make the serializer field a date we wont need this

File: mqr/utils.py [hunk 2]
```diff
@@ -70,3 +71,49 @@ def get_next_send_date():
...
+    elif 21 <= weeks_pregnant <= 25:
+        return "21-25"
+    elif 26 <= weeks_pregnant <= 30:
+        return "26-30"
+    else:
+        return None
+
+
+def get_age_bucket(age):
+    mom_age = int(age)
```
Comment (erikh360) [inline mqr/utils.py]: I think we can also make the serializer field for this an int, then we won't need this

**Review by erikh360 — COMMENTED** [review]
Nice!, just a few small comments

File: mqr/utils.py (hunk 1 above)
Comment (Buhle79) [reply mqr/utils.py]: Done

File: mqr/utils.py (hunk 2 above)
Comment (Buhle79) [reply mqr/utils.py]: Done

**Review by erikh360 — APPROVED** [review]

**Pull request merged**